setup.py

```diff
@@ -61,6 +61,7 @@
 ]
 
 install_requires = [
+    'celery>=3.1.16,<4.0.0',
```

Comment:
It doesn't work with only setting ``Flask-CeleryExt>=0.3.0`` as @egabancho suggested?